optuna/storages/redis.py

```diff
@@ -108,7 +108,7 @@ def create_new_study(self, study_name=None):
                 user_attrs={},
                 system_attrs={},
                 n_trials=0,
-                datetime_start=datetime.now(),
+                datetime_start=None,
```

Comment:
comment: In InMemoryStorage and RDBStorage, the `datetime_start` is not the time the study created but the time the first trial in the study started.